### ISSUE TITLE
Vmmem starts at boot and even after terminating it starts again by its own

### ISSUE THREAD

**Issue by AliXEagle** [microsoft/WSL#7154]
### Windows Build Number

[Version 10.0.19043.1081]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.4.72

### Distro Version

Ubuntu 20.04

### Repro Steps

Vmmem starts automatically when I boot my system and even if I terminated via PowerShell its starts again by its selfe.
I already uninstalled wsl2 and my Ubuntu distro, did a Disk Clean up to remove any temp files but it keeps starting again by itself.

### Expected Behavior

Vmmem should not start on its own. I had it on another machine and it would only start if I opened the Windows Terminal with my bash shell.

### Actual Behavior

Vmmem starts automatically when I boot my system and even if I terminated via PowerShell its starts again by its selfe.
I already uninstalled wsl2 and my Ubuntu distro, did a Disk Clean up to remove any temp files but it keeps starting again by itself.

**Comment by benhillis** [comment]
I suspect you have Linux file (something in \\\\wsl$\\) in your recent files list.  Can you try clearing your recent files list?

**Comment by AliEagle** [comment]
> I suspect you have Linux file (something in \\wsl$\) in your recent files list. Can you try clearing your recent files list?

I just did what you suggested plus, I uninstalled wsl2, Ubuntu-20.04, deleted all temps files from C:\Users\user\AppData\Local\Temp and C:\Users\alicr\AppData\Local\Packages\CanonicalGroupLimited.Ubuntu20.04onWindows_79rhkp1fndgsc

Did a Disk Clean up on both my drives 

Then did a clean install of Ubuntu-20.04 and wsl2 and after restarting...

Vmmem started running again by its own

**Comment by AliEagle** [comment]
I ended up doing the same above and installing Kali instead of Ubuntu
I guess there might be some sort of temp file somewhere very well hidden that when we install same distro as before it holds to some profile or setting that kind of restores from last one.

Not sure if its the best solution, but I leave it here in case someone else has the same issue.

It would be nice to have this revised and addressed though. 

Thanks for the help.

**Comment by sgzman** [comment]
> I suspect you have Linux file (something in \\wsl$\) in your recent files list. Can you try clearing your recent files list?

thanks, it is helpful.

**Comment by craigloewen-msft** [comment]
We'll close this for now as we don't have anything actionable for the team. This is currently by design, if there are references to files in areas like File Explorer's recent access etc. then the distro will be spun up to query that file. In the future we can explore improving this behaviour to not spin up the distro as frequently.

**Comment by omgmajk** [comment]
I have the same issue and every time it spins up it also does this, which stutters my sound in windows. I have linked my sound stuttering to wsl doing this every couple of minutes and I have no idea why or how it can be fixed. 

![hyperv](https://user-images.githubusercontent.com/10681392/125694606-fbb3e7f6-0df2-4105-96c1-7ce8cc6d0919.png)

**Comment by omgmajk** [comment]
> I suspect you have Linux file (something in \\wsl$\) in your recent files list. Can you try clearing your recent files list?

Also, I just cleared my recent files and it spun up after a wsl --shutdown automatically, also a wsl -t kali-linux has the same behaviour.

**Comment by AliXEagle** [comment]
> > I suspect you have Linux file (something in \wsl$) in your recent files list. Can you try clearing your recent files list?
> 
> Also, I just cleared my recent files and it spun up after a wsl --shutdown automatically, also a wsl -t kali-linux has the same behavior.


What worked for me after hours of trying everything I found online was to:

- Uninstall WSL completely
- Uninstall my linux distro/s (*backup as needed*)
- Cleared the recent files list (Open File Explorer>click on File>Change folder and search options>Privacy - click Clear.
- Did a Disk Cleanup (WindowsKey>type Disc Cleanup and run as Administrator>*I cleared everything in there, also had made sure to backup anything if needed* >
- Restart *at this point there should be no hyper-v or Vmmem spun*
- Clean install of wsl2
- Clean install of a different distro

This got rid of the auto-spun of hyper-v and/or Vmmem completely.

Not sure is the best solution, as it could be a pain to change distro but, luckily I only had Ubunto20 and now I'm on Kali(loving it btw).

Wish you luck.
